source/_integrations/litterrobot.markdown

```diff
@@ -34,31 +34,30 @@ The Feeder-Robot is not currently supported by this integration.
-| ----------------------------- | -------- | -------------------------------------------------------------------------------- |
-| Litter Box                    | `vacuum` | Main entity that represents a Litter-Robot unit.                                 |
-| Night Light Mode              | `switch` | When turned on, automatically turns on the night light in darker settings.       |
-| Panel Lockout                 | `switch` | When turned on, disables the buttons on the unit to prevent changes to settings. |
-| Sleep Mode Start Time         | `sensor` | When sleep mode is enabled, displays the current or next sleep mode start time.  |
-| Sleep Mode End Time           | `sensor` | When sleep mode is enabled, displays the current or last sleep mode end time.    |
-| Waste Drawer                  | `sensor` | Displays the current waste drawer level.                                         |
-| Clean Cycle Wait Time Minutes | `select` | View and select the clean cycle wait time.                                       |
-| Reset Waste Drawer            | `button` | Button to reset the waste drawer level to 0%.                                    |
+| Entity                        | Domain   | Description                                                                                                                                     |
+| ----------------------------- | -------- | ----------------------------------------------------------------------------------------------------------------------------------------------- |
+| Litter Box                    | `vacuum` | Main entity that represents a Litter-Robot unit.                                                                                                |
+| Night Light Mode              | `switch` | When turned on, automatically turns on the night light in darker settings.                                                                      |
+| Panel Lockout                 | `switch` | When turned on, disables the buttons on the unit to prevent changes to settings.                                                                |
+| Last Seen                     | `sensor` | Displays the time the unit was last seen / reported an update.                                                                                  |
+| Sleep Mode Start Time         | `sensor` | When sleep mode is enabled, displays the current or next sleep mode start time.                                                                 |
+| Sleep Mode End Time           | `sensor` | When sleep mode is enabled, displays the current or last sleep mode end time.                                                                   |
+| Status Code                   | `sensor` | Displays the [status code](https://github.com/natekspencer/pylitterbot/blob/884944b011f5fea9639b7d21d19fa3f7708e25a7/pylitterbot/enums.py#L44). |
```

Comment:
We should **never** link end-user into code to figure out status codes. That is not end-user-friendly.